osv/models.py

```diff
@@ -332,18 +332,25 @@ def _tokenize(self, value):
     value_lower = value.lower()
     return re.split(r'\W+', value_lower) + [value_lower]
 
+  def _normalize(self, name):
```

Comment:
nit: s/_normalize/_normalize_pypi/ to be clearer. (but see below for another comment which makes this no longer relevant).

Reply:
Ack. Addressed in 47438b76ae7c3b24ee97d85478bc01325d888bb6